rancid-tomatillos/src/SingleMovie.js

```diff
@@ -1,26 +1,30 @@
 import React from "react";
 import './SingleMovie.css';
+import PropTypes from "prop-types";
 
 const SingleMovie = ({ movie, displayMainPage }) => {
    return (
     <div className="single-movie-info">
-      
       <img className="single-img"src={movie[0].poster_path} />
       <div className="movie-info">
         <h2 className="single-title">{movie[0].title}</h2>
         <p>Release Date: {(movie[0].release_date)}</p>
-        {/* <p>Genre: {(this.state.movies[0].genres)}</p>
-        <p>Tagline: {(this.state.movies[0].tagline)}</p>
-        <p>Plot: {(this.state.movies[0].overiew)}</p>
-        <p>Runtime: {(this.state.movies[0].runtime)}</p> */}
+        <p>Genre: {(movie[0].genres)}</p>
```

Comment:
Is there a way to retrieve the movie without using the index number here? Be sure to DRY this section up.